external/kafka/src/main/scala/org/apache/gearpump/streaming/kafka/lib/source/AbstractKafkaSource.scala

```diff
@@ -74,11 +74,10 @@ abstract class AbstractKafkaSource(
   private lazy val kafkaClient: KafkaClient = kafkaClientFactory.getKafkaClient(config)
   private lazy val fetchThread: FetchThread = fetchThreadFactory.getFetchThread(config, kafkaClient)
   private lazy val messageDecoder = config.getConfiguredInstance(
-    KafkaConfig.MESSAGE_DECODER_CLASS_CONFIG, classOf[MessageDecoder])
-  private lazy val timestampFilter = config.getConfiguredInstance(
```

Comment:
previously filter is used to filter out old messages (e.g. timestamp < startTime) and carried out in KafkaSource implicitly. Now I think it should be defined explicitly in the following operations by users like `withAllowedLateness` in Beam API although that is not available in Gearpump API yet.